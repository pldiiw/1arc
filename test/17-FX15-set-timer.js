'use strict';

let engine = require('engine').initialize();
const setTimer = require('instructions').setTimer;

engine.data[0xB] = 147;
engine.data[0xC] = 32;

const engine1 = setTimer(engine, 0xB);
const engine2 = setTimer(engine1, 0xC);

console.log(engine.timer);
console.log(engine1.timer);
console.log(engine2.timer);
