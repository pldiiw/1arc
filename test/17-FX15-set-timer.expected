0
147
32
